module/dice/d20-roll.mjs

```diff
@@ -57,6 +54,17 @@ export default class D20Roll extends Roll {
 
   /* -------------------------------------------- */
 
+  /**
+   * Does this roll start with a d20?
+   * @type {boolean}
+   * @protected
+   */
+  get _validD20Roll() {
```

Comment:
This is my fault for not specifying originally, but it seemed like it could be a useful public method. For example, creating a `new D20Roll` and checking `roll.validD20Roll`, then performing some action. Might be more relevant now that the constructor no longer throws an error. I'll leave it up to you if you think it's worth changing before merge.